chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts

```diff
@@ -7,17 +7,20 @@ export class MockCustomerService {
 
   constructor() { }
   public generateCustomer() {
-    // TODO: create a new customer.
-    const newCust = {
-      firstName: this.generateFirstName(),
-      lastName: this.generateLastName(),
-      email: this.generateEmail(),
+    let generateCustomer;
```

Comment:
No need for this variable.  Please remove.

Reply:
Removed genCustomer